**12. Cuantos comentarios de revisión de artículos usan la palabra "mejor" (sin incluir sus
variaciones) (⭐⭐)**

In [ ]:
import pandas as pd
import numpy as np
from pydrive.auth import GoogleAuth
from pydrive.drive import GoogleDrive
from google.colab import auth
from oauth2client.client import GoogleCredentials

# Authenticate and create the PyDrive client
auth.authenticate_user()
gauth = GoogleAuth()
gauth.credentials = GoogleCredentials.get_application_default()
drive = GoogleDrive(gauth)

Para este ejercicio voy a utilizar el csv **contents**

In [ ]:
id='1SnCVFJvTsaEPrmh-PpVGH6-toBVsD6NL'
downloaded = drive.CreateFile({'id': id})
downloaded.GetContentFile('contents.csv')

Como el csv es muy grande solo voy a leer algunas columnas para ahorrar memoria

In [ ]:
contents = pd.read_csv('contents.csv', encoding='latin-1' , usecols=['title', 'id', 'revisor_username', 'revisor_comment'], \
                            dtype={'title':'category','revisor_username':'category'})

contents.head()

title  ...                                    revisor_comment
0  Wikipedia:Artículos solicitados  ...                                    Así esta mejor
1                           Andorra  ...                                  Añadir contenido
2                         Argentina  ...  Reemplazos con [[Usuario:Benjavalero/Replacer|...
3             Geografía de Andorra  ...  Reemplazos con [[Usuario:Benjavalero/Replacer|...
4            Demografía de Andorra  ...  Correcciones ortográficas con [[Usuario:Benja...

[5 rows x 4 columns]

La columna que nos interesa es **"revisor_comment"**, busco cuales comentarios contienen la palabra "mejor"

In [ ]:
contents_mejor = contents['revisor_comment'].str.contains('mejor', na = False)
contents_mejor.value_counts()

False    4121340
True       10824
Name: revisor_comment, dtype: int64

Como podemos ver hay 10824 True.

Esto significa que hay **10824** comentarios que contienen la palabra "mejor"